### PR TITLE
set count to run job

### PR DESCRIPTION
**Pull request by sathik-basha**

### CODE REVIEW

**Comment by ncb000gt** [comment]
Hey, thanks for contributing! Would you be able to update the docs and include some testing for this?

Thanks!

File: lib/cron.js [hunk 1]
```diff
@@ -590,8 +594,12 @@
 		// and does the real callback logic when it's time.
 
 		function callbackWrapper() {
+			if (self.maxJobCount === self.jobFinishedCount) {
+				self.stop();
+				return;
+			}
+			self.jobFinishedCount++;
```
Comment (ncb000gt) [inline lib/cron.js:601]: Having this here wont actually work. The reason is that this doesn't always indicate a successful execution. You'd want this line to be closer to the `self.fireOnTick()`.

**Comment by intcreator** [comment]
@sathik-basha can you add an explanation as to what this PR's goal is?

**Comment by intcreator** [comment]
closing as stale

**Pull request closed without merging**